pkg/webhook/stats_reporter.go

```diff
@@ -0,0 +1,127 @@
+package webhook
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/metrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+)
+
+const (
+	requestCountName     = "request_count"
+	requestLatenciesName = "request_latencies"
+)
+
+var (
+	requestCountM = stats.Int64(
+		requestCountName,
+		"The number of requests that are routed to webhook",
+		stats.UnitDimensionless)
```

Comment:
This measure is unnecessary. You can keep the one duration/latency measure below and expose two Views on it: one using Count and the other using Histogram.

Reply:
thanks, I didn't knew you could do that :)